pkg/scheduler/factory/factory.go

```diff
@@ -165,10 +167,6 @@ type Configurator interface {
 // configFactory is the default implementation of the scheduler.Configurator interface.
 type configFactory struct {
 	client clientset.Interface
-	// a means to list all known scheduled pods.
-	scheduledPodLister corelisters.PodLister
```

Comment:
move these fields to line 191 is not  necessary to me.

Reply:
I'll leave this to the approver.